main.py

```diff
@@ -4,7 +4,7 @@
 
 
 def main(argv: List[str]) -> None:
-    print("Hello {}".format(argv[0]))
+    print("Hello {}".format(argv[1]))
```

Comment:
Will throw an `IndexError` when no argument is provided.

Reply:
yeah you have to provide an argument